src/plugins/blocksmith/components/blocksmith/index.jsx

```diff
@@ -1041,6 +1046,16 @@ class Blocksmith extends React.Component {
           }
         }
         break;
+      case 'ArrowDown':
+        if (!nextBlock && lastLine) {
```

Comment:
@nanetteCodes the deferred calculation would be `if (!nextBlock && isOnLastLineOfBlock(editorState, FOCUS_PLUGIN_TEST_CLASS)) {`